dmutils/email.py

```diff
@@ -54,8 +54,9 @@ def send_email(to_email_addresses, email_body, subject, from_email, from_name, r
             }
             if 'DM_EMAIL_BCC_ADDRESS' in current_app.config:
```

Comment:
Could do.. Perhaps I should move all the config get-from-config lines right to the top of the method to make it clearer what config vars the method depends on?